src/scene/materials/lit-options.js

```diff
@@ -0,0 +1,147 @@
+import {
+    BLEND_NONE, GAMMA_NONE
+} from '../constants.js';
+
+/**
+ * - pass: value of {@link Layer#shaderPass} of the Layer being rendered. Must be set to the
+ * same in {@link StandardMaterialOptions#pass}.
```

Comment:
Could we only have `pass` on StandardMaterialOptions, and the setter would also set it on LitOptions? Better than having to keep two in sync